### PR TITLE
[DUOS-1976][risk=no] deprecate endpoints for DulReview, DulPreview, DulCollection, DulResultRecords

### PR DESCRIPTION
**Pull request by shaemarks**
Addresses [DUOS-1976](https://broadworkbench.atlassian.net/browse/DUOS-1976)

I wasn't able to find swagger documentation for `GET /api/election/vote/${voteId}` or the actual resource method for `GET /api/consent/${consentId}/dul`

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/assets/api-docs.yaml [hunk 1]
```diff
@@ -2111,6 +2117,7 @@ paths:
                 type: string
   /api/electionReview/{electionId}:
     get:
+      deprecated: true
```
Comment (connorlbark) [inline src/main/resources/assets/api-docs.yaml:2120]: I believe this endpoint (at ElectionReviewResource.java:59) does not have an `@Deprecated` tag.

**Review by connorlbark — CHANGES_REQUESTED** [review]

**Comment by rushtong** [comment]
> I wasn't able to find ... the actual resource method for `GET /api/consent/${consentId}/dul`

Hah - it looks like I deleted it [here](https://github.com/DataBiosphere/consent/pull/910/files#diff-44109c1a51fe73bf0340a5dad2a9951d6c12f0bfac4024c2145acd8e7cf0f84fL142), sorry about that.

**Review by rushtong — APPROVED** [review]
+1 to Connor's feedback, otherwise, 👍🏽. Please feel free to merge after that fix.

File: src/main/resources/assets/api-docs.yaml (hunk 1 above)
Comment (shaemarks) [reply src/main/resources/assets/api-docs.yaml:2120]: Thanks for catching this!

**Review by connorlbark — APPROVED** [review]

**Pull request merged**